distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -617,3 +601,15 @@ generate_column_description dialect descriptor =
    like a SQL query for the given dialect or is rather a table name.
 is_probably_a_query : Text -> Boolean
 is_probably_a_query text = (text.contains "SELECT ") || (text.contains "EXEC ")
+
+## generate_truncate_table_sql choices
```

Comment:
stray comment?

Reply:
It was supposed to be a section header for the 2 methods below.